<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="mybutton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/D:/logo/button3.png);
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="mybutton" name="btnOut">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/D:/logo/button3.png);
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
  <widget class="mybutton" name="btnQuick">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/D:/logo/button3.png);
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>快速模式</string>
   </property>
  </widget>
  <widget class="mybutton" name="btnFast">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/D:/logo/button3.png);
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>极速模式</string>
   </property>
  </widget>
  <widget class="mybutton" name="btnSlow">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/D:/logo/button3.png);
color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>慢速模式</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mybutton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
